warehouse/query-core/src/main/java/datawave/query/tables/shard/IndexStatsQueryLogic.java

```diff
@@ -63,11 +63,18 @@
     public QueryLogicTransformer getTransformer(Query settings) {
         return new AbstractQueryLogicTransformer<Object,Object>() {
             
+            long queryExecutionForCurrentPageStartTime;
```

Comment:
package private?  Perhaps make private and add a getter.

Reply:
Removed this, as well as the setter since it's not needed.